### PR TITLE
Update to Python 3; Adapt to changed ACM confirmation

### PR DESCRIPTION
**Pull request by christiandietze**
- ACM confirmation mail for the certificate change and code had to be adapted
- building the code locally with Python 2.7 did not work --> updated to 3.8
- replaced urllib with requests

### CODE REVIEW

File: lambda/cfnresponse.py [hunk 1]
```diff
@@ -15,30 +15,27 @@
...
-    responseBody['Status'] = responseStatus
-    responseBody['Reason'] = 'See the details in CloudWatch Log Stream: ' + context.log_stream_name
-    responseBody['PhysicalResourceId'] = physicalResourceId or context.log_stream_name
-    responseBody['StackId'] = event['StackId']
-    responseBody['RequestId'] = event['RequestId']
-    responseBody['LogicalResourceId'] = event['LogicalResourceId']
-    responseBody['Data'] = responseData
+    response_body = {'Status': responseStatus,
+                     'Reason': 'See the details in CloudWatch Log Stream: ' + context.log_stream_name,
+                     'PhysicalResourceId': physicalResourceId or context.log_stream_name, 'StackId': event['StackId'],
```
Comment (heroldus) [inline lambda/cfnresponse.py]:
```suggestion
                     'PhysicalResourceId': physicalResourceId or context.log_stream_name, 
                     'StackId': event['StackId'],
```
For better readability, I would suggest to have one attribute per line.

File: lambda/cfnresponse.py [hunk 2]
```diff
@@ -15,30 +15,27 @@
...
+                     'RequestId': event['RequestId'], 'LogicalResourceId': event['LogicalResourceId'],
```
Comment (heroldus) [inline lambda/cfnresponse.py]:
```suggestion
                     'RequestId': event['RequestId'], 
                     'LogicalResourceId': event['LogicalResourceId'],
```

**Review by heroldus — CHANGES_REQUESTED** [review]
Great work!

**Comment by christiandietze** [comment]
Thank you @heroldus 😄

**Review by heroldus — APPROVED** [review]
👍

**Review by clipka24 — APPROVED** [review]
LGTM 👍

**Pull request merged**